### ISSUE TITLE
Change default column for QLP light curves

### ISSUE THREAD

**Issue by avanderburg** [lightkurve/lightkurve#1083]
#### Problem description
When LightKurve reads light curves from the QLP HLSP, it pulls from a post-processed column (called KSP_FLUX) that is already high-pass-filtered. This makes it impossible to recover the original light curve and see features like stellar variability, etc. Here's a comparison of the same star from three different pipelines: 

![hlspcomparison](https://user-images.githubusercontent.com/32683658/120834889-3d6fcc00-c529-11eb-9d9d-078b466bc6bb.png)

You can see that the large modulation has been removed from QLP that is present in the other two light curves. 

As far as I can tell, there is no way to change this behavior when calling search_lightcurve. 

#### Example

search_result = lk.search_lightcurve('TIC 277554109', author='qlp')
lc = search_result.download()
lc.plot()
search_result2 = lk.search_lightcurve('TIC 277554109', author='pathos')
lc2 = search_result2.download()
lc2.plot()
search_result3 = lk.search_lightcurve('TIC 277554109', author='cdips')
lc3 = search_result3.download()
lc3.plot()

I think this can be fixed by swapping out "kspsap_flux" for "sap_flux" in this line: 
https://github.com/lightkurve/lightkurve/blob/f452a4e4752d9b277025a2ec711ec06b88230d97/src/lightkurve/io/qlp.py#L12

Alternatively, it might be useful to allow the user to specify the flux column when downloading light curves in this way. But I would argue that by default, the QLP light curves should return the sap_flux column to match the behavior of other HLSPs. 

#### Expected behavior
I would expect QLP to return the SAP_FLUX column rather than the KSPSAP_FLUX column

#### Environment

-  platform:  OSX
-  lightkurve version: 2.0.9
-  installation method: pip

**Comment by barentsen** [comment]
Thanks for the feedback @avanderburg!!

I received a few questions from confused users about this behavior, and agree it would be better to show SAP_FLUX by default to avoid non-exoplanet signals from being overlooked.  I will go ahead and open a PR to make this change.

Question: there does not appear to be a `SAP_FLUX_ERR` column in the QLP light curve files (i.e. there are no no SAP-specific flux uncertainties).  Would you recommend using the `KSPSAP_FLUX_ERR` for the SAP flux uncertainty, or would it be best to populate the uncertainties with NaN values?  (All LightCurve methods are designed to work fine with NaN errors, so this wouldn't be an issue and would be my default choice.)

Note: it is currently possible for users to select a specific flux column, but it is insufficiently documented, so I must improve the documentation for this as well.  In case anyone stumbles across this issue, I will spell out the three methods to select a specific flux column below:

## How to select the flux column in Lightkurve?

1. When downloading a light curve from MAST, pass `flux_column` to the `download()` method, e.g.:
```python
search_result = lk.search_lightcurve('TIC 277554109', author='qlp')
lc = search_result.download(flux_column="sap_flux")
```
2. When opening a local light curve file, `flux_column` can be passed to the `read()` function, e.g.:
```python
lc = lk.read("hlsp_qlp_tess_ffi_s0011-0000000277554109_tess_v01_llc.fits", flux_column="sap_flux")
```
3. *New in Lightkurve v2.10*: the `select_flux()` method has been added to enable the choice of flux column to be changed after loading the light curve. Example:
```python
lc2 = lc.select_flux("sap_flux")
```

**Comment by avanderburg** [comment]
Thanks Geert!! And thank you for the quick primer on other ways to change the flux_column! Very helpful!

In terms of the uncertainty, I checked with Chelsea and it should be fine to use the KSPSAP_FLUX_ERR. 

Thanks!
Andrew

**Comment by barentsen** [comment]
Thanks again for the feedback @avanderburg! 

I believe this issue has now been resolved via #1084, i.e., QLP light curves are now populated with `SAP_FLUX` as the default flux.  This will be included in the upcoming v2.10 release.


Demo of the new behavior:

<img width="1000" alt="Screen Shot 2021-06-04 at 3 21 45 PM" src="https://user-images.githubusercontent.com/817669/120868298-9ac74580-c548-11eb-88d9-03a88c182d32.png">

**Comment by barentsen** [comment]
Lightkurve v2.0.10 was released on pypi a few moments ago! 🎈
